libr/socket/http_server.c

```diff
@@ -72,6 +74,23 @@ R_API RSocketHTTPRequest *r_socket_http_accept (RSocket *s, int accept_timeout,
 			} else
 			if (!strncmp (buf, "Content-Length: ", 16)) {
 				content_length = atoi (buf+16);
+			} else
```

Comment:
if and else in the same line

Reply:
In fact i did for style consistency sake even though this particular part of the code is a bit inconsistent with r2 as a whole. I may then refactor a bit the even unchanged parts then ...